contract_update_last_date_invoiced/views/contract_line.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="contract_line_form_view">
+        <field name="model">contract.line</field>
+        <field
+            name="name"
+        >contract.line.form (in contract_update_last_date_invoiced)</field>
+        <field name="inherit_id" ref="contract.contract_line_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
```

Comment:
Please move the button closer to the field it will modify:

<img width="1042" height="818" alt="Image" src="https://github.com/user-attachments/assets/7afed6a7-12cf-4d73-96e4-02689daf40ef" />

You can use "arrow link" style buttons, just like in settings, so it looks better there:
<img width="1179" height="440" alt="image" src="https://github.com/user-attachments/assets/68537856-f9ed-4ac6-83e5-61b24e2e4372" />